### DISCUSSION TITLE
i have some error with loader, i dont know how to fix it

### DISCUSSION THREAD

**Discussion by minhalo** [retextjs/retext#74] [Q&A]
./node_modules/retext-emoji/index.js 25:17
Module parse failed: Identifier directly after number (25:17)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { gemoji } from 'gemoji';
| const own = {}.hasOwnProperty;
> const vs16 = 0xfe_0f;
| /** @type {Record<string, gemoji[number]>} */
|

**Comment by minhalo** [comment]
these errors  display when i import emoji from 'retext-emoji'

**Reply by wooorm** [reply]
You have an outdated Babel/Webpack/other tools. Updating them makes it work.
